### PR TITLE
adapt release action to make-release.sh changes

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
adapt release action to make-release.sh changes

### What issues does this PR fix or reference?
it's the last piece for so it fixes https://github.com/devfile/devworkspace-operator/issues/520

### Is it tested? How?
I tested it in my fork https://github.com/sleshchenko/devworkspace-operator/actions

### PR Checklist

- [ ] E2E tests pass (when PR is ready, comment `/test v7-devworkspaces-operator-e2e, v7-devworkspace-happy-path` to trigger)
    - [ ] `v7-devworkspaces-operator-e2e`: DevWorkspace e2e test
    - [ ] `v7-devworkspace-happy-path`: DevWorkspace e2e test

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -52,22 +84,39 @@ jobs:
           pip install yq
 
           cd ${GITHUB_WORKSPACE}
-      - name: Build image, generate deployments and push git tag
+
+      - name: Release
+        env:
+          DWO_REPO: git@github.com:sleshchenko/devworkspace-operator
```
Comment (JPinkney) [inline .github/workflows/release.yml:90]: Looks like you still have your repo and images setup, but other then that it looks good to me

Comment (sleshchenko) [reply .github/workflows/release.yml]: Good catch!

**Review by JPinkney — APPROVED** [review]

**Pull request merged**